src/main/resources/com/eprosima/uxr/idl/templates/TypesSource.stg

```diff
@@ -83,25 +87,25 @@ enum_type(ctx, parent, enum) ::= <<>>
 // ========================================================================================
 member_serialization(ctx, typecode, name, originName) ::= <<
 $if(typecode.isType_c)$
-(void) ucdr_serialize_uint32_t(writer, (uint32_t)topic->$name$);
+su ucdr_serialize_uint32_t(writer, (uint32_t)topic->$name$);
```

Comment:
@pablogs9 Is this  a typo?

Reply:
Let me check

Reply:
Fixed here: https://github.com/eProsima/Micro-XRCE-DDS-Gen/pull/49

We are running the CI based on the IDL-Parser repo tests, probably this member type is not being used.